### PR TITLE
runTheMatrix.py: Change confusing message about number of threads.

### PR DESCRIPTION
**Pull request by gartung**
"Running in 4 thread(s)"  becomes
"Running up to 4 concurrent jobs, each with 1 thread per process"

Concurrent jobs is the number of python threads used, one per workflow, not to be confused with the number of TBB threads cmsRun uses.

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Before submitting your pull requests, make sure you followed this checklist:
- verify that the PR is really intended for the chosen branch
- verify that changes follow [CMS Naming, Coding, And Style Rules](http://cms-sw.github.io/cms_coding_rules.html)
- verify that the PR passes the basic test procedure suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html)

### CODE REVIEW

**Comment by kpedro88** [comment]
please test

**Comment by gartung** [comment]
please test

File: Configuration/PyReleaseValidation/python/MatrixRunner.py [hunk 1]
```diff
@@ -40,10 +40,11 @@ def runTests(self, opt):
         noRun=(self.maxThreads==0)
         if noRun:
             print('Not running the wf, only creating cfgs and logs')
-            print('resetting to default number of threads')
             self.maxThreads=4
+            print('resetting to default number of process threads = %s' %  self.maxThreads)
 
-        print('Running in %s thread(s)' % self.maxThreads)
+        print('Running in %s process thread(s)' % self.maxThreads)
+        print('Running in %s thread(s) per process thread' % self.nThreads)
```
Comment (fwyzard) [inline Configuration/PyReleaseValidation/python/MatrixRunner.py:47]:
```suggestion
print('Running %s %s, each with %s thread%s per process' % (self.maxThreads, 'concurrent jobs' if self.maxThreads > 1 else 'job', self.nThreads, 's' if self.nThreads > 1 else ''))
```

**Comment by kpedro88** [comment]
@gartung please update the PR description to be consistent

**Comment by kpedro88** [comment]
please test

**Comment by gartung** [comment]
abort

**Comment by gartung** [comment]
please test

**Comment by kpedro88** [comment]
+upgrade

**Comment by chayanit** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**